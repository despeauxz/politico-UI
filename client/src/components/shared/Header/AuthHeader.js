import React, { Fragment } from 'react';

const AuthHeader = () => {
    return (
        <Fragment>
            <div className="border-b flex px-6 py-2 items-center flex-none">
                <div className="flex flex-col">
                    <h3 className="text-grey-darkest mb-1 font-extrabold">#Politico</h3>
                </div>
                <div className="mx-auto md:block">
                    <div className="relative">
                        <input type="search" placeholder="Search" className="appearance-none border border-grey rounded-lg pl-8 pr-4 py-2" />
                    </div>
                </div>
            </div>
        </Fragment>
    );
};

export default AuthHeader;
