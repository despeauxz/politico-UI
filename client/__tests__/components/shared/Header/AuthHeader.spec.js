import React from 'react';
import AuthHeader from '@components/shared/Header/AuthHeader';
import toJson from 'enzyme-to-json';


describe('<AuthHeader />', () => {
    it('render the Auth header component', () => {
        const shallowWrapper = shallow(<AuthHeader />);

        expect(toJson(shallowWrapper)).toMatchSnapshot();
        expect(shallowWrapper.find('AuthHeader')).toBeTruthy();
    });
});
