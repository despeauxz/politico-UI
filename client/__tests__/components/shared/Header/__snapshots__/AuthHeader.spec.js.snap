// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthHeader /> render the Auth header component 1`] = `
<Fragment>
  <div
    className="border-b flex px-6 py-2 items-center flex-none"
  >
    <div
      className="flex flex-col"
    >
      <h3
        className="text-grey-darkest mb-1 font-extrabold"
      >
        #Politico
      </h3>
    </div>
    <div
      className="mx-auto md:block"
    >
      <div
        className="relative"
      >
        <input
          className="appearance-none border border-grey rounded-lg pl-8 pr-4 py-2"
          placeholder="Search"
          type="search"
        />
      </div>
    </div>
  </div>
</Fragment>
`;
